src/components/common/styles.js

```diff
@@ -229,48 +215,53 @@ export const StyledAnimatedStacks = styled.ul`
   overflow: hidden;
 `;
 
+/* landing page animation */
+export const StyledWave = keyframes`
```

Comment:
Alright, I see that this is used in all three landing pages. I remember suggesting this before, but I think we should remove the login/signup pages and move that functionality into the Landing.js (with appropriate child components for the login and signup forms). This way there's less duplicated code and we can move most of these styles into Landing.js